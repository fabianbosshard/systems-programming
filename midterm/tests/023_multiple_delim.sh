exec "$1" format='@a -> @b' delim== tests/lcov_config delim=_ tests/lcov_config
