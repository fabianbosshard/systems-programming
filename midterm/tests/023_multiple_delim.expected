genhtml_css_file  ->  gcov.css
genhtml_dark_mode  ->  1
genhtml_header  ->  Coverage report for my project
genhtml_footer  ->  My footer text
genhtml_hi_limit  ->  90
genhtml_med_limit  ->  75
ignore_errors  ->  empty,mismatch
max_message_count  ->  100
stop_on_error  ->  1
rtl_file_extensions  ->  v,vh,sv,vhd?
c_file_extensions  ->  h,c,i,C,H,I,cpp,hpp,icc,cc,hh,cxx,hxx
filter_lookahead  ->  5
filter_bitwise_conditional  ->  1
genhtml_line_field_width  ->  12
genhtml_branch_field_width  ->  16
genhtml_owner_field_width  ->  20
genhtml_age_field_width  ->  5
genhtml_overview_width  ->  80
genhtml_nav_resolution  ->  4
genhtml_nav_offset  ->  10
genhtml_keep_descriptions  ->  0
genhtml_no_prefix  ->  0
genhtml_no_source  ->  0
genhtml_num_spaces  ->  8
genhtml_highlight  ->  0
genhtml_legend  ->  0
genhtml_html_prolog  ->  FILE
genhtml_html_epilog  ->  FILE
genhtml_html_extension  ->  html
genhtml_html_gzip  ->  1
genhtml_sort  ->  1
genhtml_hierarchical  ->  1
genhtml_flat_view  ->  1
genhtml_show_navigation  ->  1
genhtml_show_function_proportion  ->  0
genhtml_charset -> UTF-8
genhtml_desc_html -> 0
genhtml_precision -> 1
genhtml_missed -> 1
suppress_function_aliases  ->  1
derive_function_end_line  ->  1
derive_function_end_line  ->  1
trivial_function_threshold  ->  5
demangle_cpp  ->  c++filt
genhtml_demangle_cpp_tool  ->  c++filt
genhtml_demangle_cpp_params  ->  ""
geninfo_gcov_tool  ->  gcov
geninfo_adjust_testname  ->  0
forget_testcase_names  ->  0
checksum  ->  1
geninfo_external  ->  1
geninfo_compat_libtool  ->  0
geninfo_gcov_all_blocks  ->  1
geninfo_unexecuted_blocks  ->  0
geninfo_compat  ->  libtool
geninfo_adjust_src_path  ->  /tmp/build 
geninfo_auto_base  ->  1
geninfo_intermediate  ->  auto
no_exception_branch  ->  0
lcov_insmod_tool  ->  /sbin/insmod
lcov_modprobe_tool  ->  /sbin/modprobe
lcov_rmmod_tool  ->  /sbin/rmmod
lcov_tmp_dir  ->  /tmp
lcov_list_full_path  ->  0
lcov_list_width  ->  80
lcov_list_truncate_max  ->  20
function_coverage  ->  1
branch_coverage  ->  0
lcov_fail_under_lines  ->  97.5
genhtml_annotate_script  ->  path_to_my_executable
genhtml_criteria_script  ->  path_to_my_executable
criteria_callback_data  ->  date,owner
criteria_callback_levels  ->  top,directory,file
genhtml_date_bins  ->  7
genhtml_date_bins  ->  30,180
include  ->  my/files/*.c
include  ->  my/files/*.h
exclude  ->  your/files/*.c
exclude  ->  your/files/*.h
substitute  ->  s#/.libs##g
substitute  ->  s#/tmp/build#/usr/src#g
omit_lines  ->  .+?//\s*MY_LINE_EXCLUDE_MARKER\s*
omit_lines  ->  .+?//\s*MY_PROJECT_EXCLUDE_MARKER\s*
erase_functions  ->  main
erase_functions  ->  debug)_.*\(int,\s*int\)
version_script  ->  path_to_my_executable
lcov_json_module  ->  auto
parallel  ->  1
memory  ->  1024
split_char  ->  ,
case_insensitive  ->  0
lcov_excl_line  ->  LCOV_EXCL_LINE
lcov_excl_br_line  ->  LCOV_EXCL_BR_LINE
lcov_excl_exception_br_line  ->  LCOV_EXCL_EXCEPTION_BR_LINE
lcov_excl_start  ->  LCOV_EXCL_START
lcov_excl_stop  ->  LCOV_EXCL_STOP
lcov_excl_br_start  ->  LCOV_EXCL_BR_START
lcov_excl_br_stop  ->  LCOV_EXCL_BR_STOP
lcov_excl_exception_br_start  ->  LCOV_EXCL_EXCEPTION_BR_START
lcov_excl_exception_br_stop  ->  LCOV_EXCL_EXCEPTION_BR_STOP
genhtml -> css
genhtml -> dark
genhtml -> header = Coverage report for my project
genhtml -> footer = My footer text
genhtml -> hi
genhtml -> med
ignore -> errors = empty,mismatch
max -> message
stop -> on
rtl -> file
c -> file
filter -> lookahead = 5
filter -> bitwise
genhtml -> line
genhtml -> branch
genhtml -> owner
genhtml -> age
genhtml -> overview
genhtml -> nav
genhtml -> nav
genhtml -> keep
genhtml -> no
genhtml -> no
genhtml -> num
genhtml -> highlight = 0
genhtml -> legend = 0
genhtml -> html
genhtml -> html
genhtml -> html
genhtml -> html
genhtml -> sort = 1
genhtml -> hierarchical = 1
genhtml -> flat
genhtml -> show
genhtml -> show
genhtml -> charset=UTF-8
genhtml -> desc
genhtml -> precision=1
genhtml -> missed=1
suppress -> function
derive -> function
derive -> function
trivial -> function
demangle -> cpp = c++filt
genhtml -> demangle
genhtml -> demangle
geninfo -> gcov
geninfo -> adjust
forget -> testcase
checksum = 1 -> 
geninfo -> external = 1
geninfo -> compat
geninfo -> gcov
geninfo -> unexecuted
geninfo -> compat = libtool=on, hammer=auto, split
geninfo -> adjust
geninfo -> auto
geninfo -> intermediate = auto
no -> exception
lcov -> insmod
lcov -> modprobe
lcov -> rmmod
lcov -> tmp
lcov -> list
lcov -> list
lcov -> list
function -> coverage = 1
branch -> coverage = 0
lcov -> fail
genhtml -> annotate
genhtml -> criteria
criteria -> callback
criteria -> callback
genhtml -> date
genhtml -> date
include = my/files/*.c -> 
include = my/files/*.h -> 
exclude = your/files/*.c -> 
exclude = your/files/*.h -> 
substitute = s#/.libs##g -> 
substitute = s#/tmp/build#/usr/src#g -> 
omit -> lines = .+?//\s*MY
omit -> lines = .+?//\s*MY
erase -> functions = main
erase -> functions = debug)
version -> script = path
lcov -> json
parallel = 1 -> 
memory = 1024 -> 
split -> char = ,
case -> insensitive = 0
lcov -> excl
lcov -> excl
lcov -> excl
lcov -> excl
lcov -> excl
lcov -> excl
lcov -> excl
lcov -> excl
lcov -> excl
